src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/01 Executors.md

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Executors'
+---
+
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
+
+[Executors](/using-k6/scenarios/executors) are the workhorses of the k6 execution engine. Each one
+schedules VUs and iterations differently, and you'll choose one depending on the type of traffic you
+want to model to test your services.
+
+Possible values for `executor` are the executor name separated by hyphens.
+
+| Name                                                                         | Value                   | Description                                                                                                                                        |
+| ---------------------------------------------------------------------------- | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Shared iterations](/using-k6/scenarios/executors/shared-iterations)         | `shared-iterations`     | A fixed amount of iterations are<br/> "shared" between a number of VUs.                                                                            |
+| [Per VU iterations](/using-k6/scenarios/executors/per-vu-iterations)         | `per-vu-iterations`     | Each VU executes an exact number of iterations.                                                                                                    |
+| [Constant VUs](/using-k6/scenarios/executors/constant-vus)                   | `constant-vus`          | A fixed number of VUs execute as many<br/> iterations as possible for a specified amount of time.                                                  |
+| [Ramping VUs](/using-k6/scenarios/executors/ramping-vus)                     | `ramping-vus`           | A variable number of VUs execute as many<br/> iterations as possible for a specified amount of time.                                               |
+| [Constant Arrival Rate](/using-k6/scenarios/executors/constant-arrival-rate) | `constant-arrival-rate` | A fixed number of iterations are executed<br/> in a specified period of time.                                                                      |
+| [Ramping Arrival Rate](/using-k6/scenarios/executors/ramping-arrival-rate)   | `ramping-arrival-rate`  | A variable number of iterations are <br/> executed in a specified period of time.                                                                  |
+| [Externally Controlled](/using-k6/scenarios/executors/externally-controlled) | `externally-controlled` | Control and scale execution at runtime<br/> via [k6's REST API](/misc/k6-rest-api) or the [CLI](https://k6.io/blog/how-to-control-a-live-k6-test). |
```

Comment:
Why is this repeated here and on the Scenarios page? I would remove this page and leave the executor summary only on the Scenarios page.

Reply:
There's no good way to do this with our current setup in gatsby unfortunately. Each item in the sidebar requires it's own subpage or the menu generator goes bananas. u_u Will think of this is a future improvement.